blocks/init/src/Blocks/components/pagination/pagination-style.scss

```diff
@@ -0,0 +1,82 @@
+.pagination {
+	--pagination-scoped-radius-size: calc(var(--base-font-size) * 2.7rem);
+	--pagination-scoped-color: var(--global-colors-black);
+	--pagination-scoped-background-color: transparent;
+
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+
+	@include media(tablet up) {
+		justify-content: center;
+	}
+
+	> * {
+		margin: 0 calc(var(--base-font-size) * 0.5rem);
+		font-size: calc(var(--base-font-size) * 1.7rem);
+		line-height: 1.5;
+		color: var(--pagination-scoped-color);
+		transition: color 0.3s ease-out;
+
+		span {
+			width: var(--pagination-scoped-radius-size);
+			height: var(--pagination-scoped-radius-size);
+			border-radius: calc(var(--base-font-size) * 0.4rem);
+			display: flex;
+			align-items: center;
+			justify-content: center;
+			background-color: var(--pagination-scoped-background-color);
+		}
+	}
+
+	.page-numbers {
+		display: none;
+
+		@include media(tablet up) {
+			display: block;
+		}
+	}
+
+	.prev,
```

Comment:
These should follow BEM. Please check if there's a way to add custom classes to `paginate_links`.